packages/golub/bin/golub.dart

```diff
@@ -4,7 +4,7 @@
 
 import 'dart:io' show exit;
 
-import 'package:pigeon/src/pigeon_cl.dart';
+import 'package:golub/src/pigeon_cl.dart';
```

Comment:
all files which contain `pigeon` reference should be renamed

Reply:
https://github.com/Yobari-Timeliners/golub/issues/105